src/@context/Profile.tsx

```diff
@@ -141,6 +141,7 @@ function ProfileProvider({
           poolShares
         )
       } catch (error) {
+        setIsPoolSharesLoading(false)
```

Comment:
theoretically™, the `finally` is reached on both, success and `catch`ed error so `setIsPoolSharesLoading(false)` should fire in any case. But I guess this is one of those weirdnesses you encountered? But if not we could remove that extra `setIsPoolSharesLoading(false)` here

Reply:
yes, u're right. I'l fix it in a moment